xml/System.Text.Json/JsonElement.xml

```diff
@@ -1515,7 +1515,7 @@ This method does not parse the contents of a JSON string value.
 
 This method does not parse the contents of a JSON string value.
 
-On .NET Core, this method does not return `false` for values larger than <xref:System.Single.MaxValue?displayProperty=nameWithType> or smaller than <xref:System.Single.MinValue?displayProperty=nameWithType>). Instead, it returns `false` and assigns <xref:System.Single.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Single.NegativeInfinity?displayProperty=nameWithType> to the `value` argument.
+On .NET Core, this method does not return `false` for values larger than <xref:System.Single.MaxValue?displayProperty=nameWithType> or smaller than <xref:System.Single.MinValue?displayProperty=nameWithType>). Instead, it returns <see langword="true"> and assigns <xref:System.Single.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Single.NegativeInfinity?displayProperty=nameWithType> to the `value` argument.
```

Comment:
@carlossanlop - any ideas what caused the typo? Was these hand-written or does it point to a potential porting tool bug?

These were added in https://github.com/dotnet/dotnet-api-docs/pull/2471.